src/css/settings.sizes.css

```diff
@@ -64,5 +64,5 @@
   --size-h-screen: 100vh;
   --size-w-screen: 100vw;
 
-  --size-scroll-top: 5.25rem; /* 84px */
+  --size-scroll-offset: 5.25rem; /* 84px */
```

Comment:
Would perhaps add a comment here `/* 84px — Header height */`? It's always going to be tricky to know that this value is important and if it changes, header should change. I know we all hate magic numbers, but t'is what it is.

Or sometimes I like a cheeky custom prop to give us the chance to inject a bit of meaning:

```css
--header-height: 5.25rem; /* 84px */
--size-scroll-offset: var(--header-height);
```

Reply:
Yeah, fair point.

I've added the custom prop.